public/styles.css

```diff
@@ -96,6 +96,74 @@ body {
   transition-duration: 0.3s;
 }
 
+.fixed-center {
+  top: 50%;
+	left: 50%;
+	transform: translate(-50%, -50%);
+	position: fixed;
+	z-index: 4;
```

Comment:
also not sure why github's diff is weird about the blank space in this file, looks like it's fine in my text editor locally. If you use vscode, maybe just alt + shift + f in this file to make it all consistent

Reply:
looks like I accidentally indented part of that file with tabs. just fixed it